client/app/components/sections/Topbar/Topbar.css

```diff
@@ -54,16 +54,29 @@
     }
   }
 
+  & button {
+    border-radius: 0;
+  }
+
   & .topbarLogo {
     max-height: var(--Topbar_logoMaxHeightMobile);
-    max-width: var(--Topbar_textLogoMaxWidth);
+    max-width: 70%;
```

Comment:
That's exactly what it is, and it's on purpose.

The max-width is for the text logo in the mobile breakpoint where we want to fit as much text in the Topbar as we can.